cfme/tests/automate/test_namespace.py

```diff
@@ -94,3 +95,32 @@ def test_duplicate_namespace_disallowed(request, parent_namespace):
         parent_namespace.namespaces.create(
             name=ns.name,
             description=ns.description)
+
+
+@pytest.mark.tier(2)
+def test_namespace_name_wrong(domain):
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: medium
+        caseposneg: negative
+        initialEstimate: 1/60h
+        tags: automate
+        testSteps:
+            1. Navigate to Automation> Automate> Explorer
+            2. Try to create namespace with name `Dummy Domain` (I put space which is invalid)
+        expectedResults:
+            1.
```

Comment:
Remove Empty step 1 .

Reply:
@sshveta 
As I see [suggestions](https://github.com/ManageIQ/integration_tests/pull/8443#discussion_r259077545) from @jawatts @mshriver .
This is the way of writing ```expectedResults```.